app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/health/CPUMonitorWorker.kt

```diff
@@ -44,45 +36,19 @@ class CPUMonitorWorker(
     @Inject
     lateinit var deviceShieldPixels: DeviceShieldPixels
 
```

Comment:
Incidentally, better use the `dispatcherProvider`
```suggestion

    @Inject
    lateinit var dispatcherProvider: DispatcherProvider

```